### PR TITLE
[Feature] CX-5282 Convert crossDevice flow components

### PR DESCRIPTION
**Pull request by josokinas**
Converting crossDevice flow components.

### CODE REVIEW

File: src/components/crossDevice/CrossDeviceSubmit/style.scss [hunk 1]
```diff
@@ -1,9 +1,9 @@
-@import (less) "../../Theme/constants.css";
+@import '../Theme/constants';
```
Comment (kopijunkie) [inline src/components/crossDevice/CrossDeviceSubmit/style.scss:1]: I think this needs to be `'../../Theme/constants';` due to the level of the folders

File: src/components/crossDevice/Intro/style.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@import (less) "../../Theme/constants.css";
+@import '../Theme/constants';
```
Comment (kopijunkie) [inline src/components/crossDevice/Intro/style.scss:1]: Same here.

File: src/components/crossDevice/MobileConnected/style.scss [hunk 3]
```diff
@@ -1,11 +1,11 @@
-@import (less) "../../Theme/constants.css";
+@import '../Theme/constants';
```
Comment (kopijunkie) [inline src/components/crossDevice/MobileConnected/style.scss:1]: Same here also.

File: src/components/crossDevice/MobileNotificationSent/style.scss [hunk 4]
```diff
@@ -1,10 +1,10 @@
-@import (less) "../../Theme/constants.css";
+@import '../Theme/constants';
```
Comment (kopijunkie) [inline src/components/crossDevice/MobileNotificationSent/style.scss:1]: And here.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1086-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1086-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1086-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**